app-web/plugins/gatsby-source-github-all/utils/plugins.js

```diff
@@ -186,8 +186,13 @@ const markdownResourceTypePlugin = (extension, file) => {
 const externalLinkUnfurlPlugin = async (extension, file) => {
   // does file have a resource path and is it a valid url?
   if (file.metadata.resourcePath && validUrl.isUri(file.metadata.resourcePath)) {
-    const { body: html, url } = await got(file.metadata.resourcePath);
-    const metadata = await metascraper({ html, url });
+    const browser = await puppeteer.launch();
```

Comment:
it is probably super inefficient to have a new browser loaded for every file. A browser should be instantiated outside and closed after the entire source routine is complete. This can be done for issue https://taiga.pathfinder.gov.bc.ca/project/lukegonis-developergovbcca/issues?status=584,585,586,588&q